src/translations/translations.en.js

```diff
@@ -213,6 +213,64 @@ export const translations = {
   //#region cc-doc-list
   'cc-doc-list.error': `An error occurred while loading documentation`,
   //#endregion
+  //#region cc-domain-management
+  'cc-domain-management.certif.automated': () => sanitize`Whether you use <code>cleverapps.io</code> or your own domain names with applications hosted by Clever Cloud, a Let's Encrypt certificate is automatically issued and renewed for HTTPS/TLS access. No action is required from you, this is all automated. For specific cases, refer to <a href="https://developers.clever-cloud.com/doc/administrate/ssl/">Installing TLS Certificates</a>.`,
+  'cc-domain-management.certif.custom': () => sanitize`You can provide your own certificate by using the <a href="https://api.clever-cloud.com/v2/certificates/new">Clever Cloud Certificate Manager</a>.`,
+  'cc-domain-management.certif.heading': `Secure your application`,
+  'cc-domain-management.dns.a.desc': () => sanitize`<p>If you choose to use <code>A</code> records, you'll need to update them yourself.</p>
+  <p>Keep an eye on our <a href=https://developers.clever-cloud.com/changelog/">changelog</a> or check our <a href="https://developers.clever-cloud.com/api/v4/#load-balancers">v4 API documentation</a> for this.</p>`,
+  'cc-domain-management.dns.a.heading': `A records`,
+  'cc-domain-management.dns.cname.desc': () => sanitize`<p>Using a <code>CNAME</code> record is highly recommended. This automatically keeps your configuration up to date.</p>`,
```

Comment:
question: we don't mention something like "not for apex domains", maybe it's too much?